### ISSUE TITLE
onDidChangeCursorPosition sends "textChanged" as false when backspace is pressed

### ISSUE THREAD

**Issue by mauricioszabo** [atom/atom#11870]
### Prerequisites
* Atom 1.7.3
* Adds a callback to `onDidChangeCursorPosition`

### Description
When pressing backspace or delete, `textChanged` property of event is set to `false`.

Tested on Ubuntu 16.04, with Atom 1.7.3

**Comment by mauricioszabo** [comment]
This issue still occurs on Atom `1.18.0`. It doesn't matter that it was "marked as stale" - it IS a bug, and should be corrected - it impacts directly on plug-ins developers...

**Comment by chrispahm** [comment]
Just ran into this issue as well on Atom `1.27.1` on Windows.

**Comment by lierdakil** [comment]
Still a thing in Atom 1.32.0.

**Comment by mauricioszabo** [comment]
I don't think this will ever be addressed: it's been 2 years already, and no new comments were made... nor even a guide on where to begin looking, so someone interested in helping should address this.

**Comment by mauricioszabo** [comment]
1. Verify that you can still reproduce the issue in the latest version of Atom
  Yes, it still happens, is still reproducible

2.1 What version of Atom you reproduced the issue on
  Atom 1.40.1

2.2 What OS and version you reproduced the issue on
  Ubuntu 18.04

2.3 What steps you followed to reproduce the issue
  Used the devtools' console to run the following code:
```js
atom.workspace.getActiveTextEditor().onDidChangeCursorPosition(console.log)
```

Check `textChanged` property, and see it is `false` when backspace is pressed.

**Comment by mauricioszabo** [comment]
Verify that you can still reproduce the issue in the latest version of Atom
Yes, it still happens, is still reproducible as of Atom 1.53.0

2.1 What version of Atom you reproduced the issue on
Atom 1.53.0

2.2 What OS and version you reproduced the issue on
Ubuntu 18.04

2.3 What steps you followed to reproduce the issue
Used the devtools' console to run the following code:

atom.workspace.getActiveTextEditor().onDidChangeCursorPosition(console.log)
Check textChanged property, and see it is false when backspace is pressed.
